frontend-test/unit/smaaResultsServiceSpec.js

```diff
@@ -454,5 +454,87 @@ define([
         expect(result).toEqual(expectedResult);
       });
     });
+
+    describe('hasNoStochasticMeasurements', function() {
+      it('should return false if there is a performance that has a distribution that is not exact', function() {
+        const aggregateState = {
+          problem: {
+            performanceTable: [{
+              performance: {
+                distribution: {
+                  type: 'not exact'
+                }
+              }
+            }]
+          }
+        };
+        const result = resultsService.hasNoStochasticMeasurements(aggregateState);
+        expect(result).toBeFalsy();
+      });
+
+      it('should return true if there is no performance that has a distribution that is not exact', function() {
+        const aggregateState = {
```

Comment:
A helper to set up the base aggregate state for this test would cut down a lot on duplicate lines